actions/java-gradle-release/action.yaml

```
@@ -109,9 +109,11 @@ runs:
         tag: ${{ steps.evaluate-version.outputs.release-version }}
         changelog-file: CHANGELOG.md
 
-    - name: Commit and push changes including the bump config file
-      run: |
-        git add CHANGELOG.md
-        git commit -m "Changelog for version ${{ steps.evaluate-version.outputs.release-version }}"
-        git push --follow-tags origin ${{ github.event.repository.default_branch }}
-      shell: bash
+    - name: Commit and push CHANGELOG.md file
+      uses: bakdata/ci-templates/actions/commit-and-push@1.50.2
```

please bump all actions to 1.50.3
```suggestion
      uses: bakdata/ci-templates/actions/commit-and-push@1.50.3
```